### PR TITLE
Integrate new code examples for button.

### PR DESCRIPTION
**Pull request by FabienLelaquais**
+ RefMan improvement on classes.

### CODE REVIEW

File: docs/refmans/gui/viselements/generic/button.md_template [hunk 1]
```diff
@@ -2,44 +2,105 @@ A control that can trigger a function when pressed.
...
+## Action as a lambda function {data-source="gui:doc/examples/controls/button_lambda.py"}
+
+A callback function can be defined as a lambda function to handle button actions.
+
+Below is an example of defining a button using a lambda function for the [*on_action*](#p-on_action)
+property:
+!!! taipy-element
+    default=Button Label
+    id=button1
+    on_action:f=lambda s, i, p: s.assign("text", f"button '{i}': '{p.get('action')}'.")
```
Comment (FredLL-Avaiga) [inline docs/refmans/gui/viselements/generic/button.md_template:54]: More descriptive parameter names? 
State, id, payload

File: docs/refmans/gui/viselements/generic/button.md_template [hunk 2]
```diff
@@ -2,44 +2,105 @@ A control that can trigger a function when pressed.
...
+
+The lambda function takes the same parameters as a standard `on_action` callback:
+
+- *s* (the state object),
```
Comment (FredLL-Avaiga) [inline docs/refmans/gui/viselements/generic/button.md_template:58]: Ok ...

File: docs/refmans/gui/viselements/generic/button.md_template [hunk 3]
```diff
@@ -2,44 +2,105 @@ A control that can trigger a function when pressed.
...
+## Using an icon {data-source="gui:doc/examples/controls/button_icon.py"}
+
+A button can contain text, an icon, or both.
+
+To add an icon to a button, you must first create an instance of the `Icon^` object, which
+represents the image and its associated label (text).
+
+To add an icon to a button, you must create an `Icon^` instance:
+```python
+icon = Icon("charles-avatar.png", "Charles")
```
Comment (FredLL-Avaiga) [inline docs/refmans/gui/viselements/generic/button.md_template:79]: ./charles-avatar.png ?

**Review by FredLL-Avaiga — APPROVED** [review]

File: docs/refmans/gui/viselements/generic/button.md_template (hunk 1 above)
Comment (FabienLelaquais) [reply docs/refmans/gui/viselements/generic/button.md_template:54]: That would make the line a bit too long (in the Builder construct) in my opinion...

File: docs/refmans/gui/viselements/generic/button.md_template (hunk 3 above)
Comment (FabienLelaquais) [reply docs/refmans/gui/viselements/generic/button.md_template:79]: No need for that. I tested it.
And that's identical (now) to the code example in the repo.

**Pull request merged**